src/ConfigJson/Config.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+declare(strict_types=1);
+
+namespace ConfigCat\ConfigJson;
+
+use UnexpectedValueException;
+
+/**
+ * Represents the root JSON keys of a ConfigCat config_v6.json file.
+ */
+abstract class Config
+{
+    /**
+     * Preferences of the config.json, mostly for controlling the redirection behaviour of the SDK.
+     */
+    public const PREFERENCES = 'p';
+
+    /**
+     * Segment definitions for re-using segment rules in targeting rules.
+     */
+    public const SEGMENTS = 's';
+
+    /**
+     * Setting definitions.
+     */
+    public const SETTINGS = 'f';
+
+    /**
+     * @param array<string, mixed> $config
+     *
+     * @internal
+     */
+    public static function fixup(array &$config): void
```

Comment:
I'd prefer a more descriptive function name here: like for example inlineSaltAndSegments